code/modules/research/nanites/nanite_programs/buffing.dm

```diff
@@ -43,15 +43,15 @@
 	. = ..()
 	if(ishuman(host_mob))
 		var/mob/living/carbon/human/H = host_mob
-		H.physiology.armor.melee += 50
```

Comment:
Damn, dermal hardening and dermal refractive surface were the main reasons why I make nanoboost viruses as a virologist, it's gonna be sad to see their strength basically halved. They'll still be good, just not really worth it outside of specially-crafted nanoboost virus+full aug setups.

Reply:
The alternative was making the entire armor system of the game completely and utterly gutted.